crates/xilem_html/web_examples/mathml_svg/index.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<title>Pythagorean theorem</title>
```

Comment:
Yeah I was interested, what the minimum required valid HTML is, and it seems, that just a `<title>` is required.

But this will be transformed anyway by trunk, so the generated html looks more typical.